demo/src/screens/componentScreens/PickerScreen.tsx

```diff
@@ -289,6 +296,39 @@ export default class PickerScreen extends Component {
             style={{alignSelf: 'flex-start'}}
             onPress={() => this.picker.current?.openExpandable?.()}
           />
+
+          <Text text60 marginT-s5>
+            Different Field Types
+          </Text>
+          <Text text80 marginB-s5>(Form/Filter/Settings)</Text>
```

Comment:
Add the `$textDefault` modifier for the dark mode support

Reply:
Same answer as before